iroha/src/sumeragi.rs

```diff
@@ -995,6 +1021,12 @@ pub mod message {
     }
 
     impl BlockSigned {
+        /// # Errors
+        /// Asserts specific instruction number of instruction in transaction constraint
+        pub fn check_instruction_len(&self, max_instruction_number: usize) -> Result<()> {
+            self.block.check_instruction_len(max_instruction_number)
+        }
```

Comment:
Can we put this check into validation stage and have only one type with it?

Reply:
Didn't get what you mean. I think it might be handy to have this check for every type of block and transaction.